docs/element-definitions.md

```diff
@@ -210,8 +219,8 @@ Note: At the moment only the HTTP protocol is supported.
 
         Definition of any input message-body attribute for this transition.
 
-    - `contentTypes` (array[String]) - A collection of content types that MAY be used for the transition.
-- `content` (array)
+    - `contentTypes` (Array Element[[String Element][]]) - A collection of content types that MAY be used for the transition.
+- `content` (array, fixed-type)
```

Comment:
I think we can disregard my existing comment. Content may be array, I would remove `fixed-type` though, commented on that further above.